src/browser.qml

```diff
@@ -113,7 +113,7 @@ ApplicationWindow {
                 }
             }
 
-            width: height
+            width: page ? (page.isPortrait ? Screen.width : Screen.height) : 0
             height: Math.max(Screen.height, Screen.width)
```

Comment:
Not really. Background is not rotating along with the page.

If you want to give a test ride... try with proposed fix landscape testwebprompts.html